Trappist/src/Promact.Trappist.Web/Migrations/20170310033212_migration-csq-1.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.EntityFrameworkCore.Migrations;
+using Microsoft.EntityFrameworkCore.Metadata;
+
+namespace Promact.Trappist.Web.Migrations
+{
+    public partial class migrationcsq1 : Migration
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Rename class 'migrationcsq1' to match camel case naming rules, consider using 'Migrationcsq1'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://server12:9005/coding_rules#rule_key=csharpsquid%3AS101)